examples/wuerstchen/text_to_image/requirements.txt

```diff
@@ -0,0 +1,7 @@
+accelerate>=0.16.0
+torchvision
+transformers>=4.25.1
+wandb
```

Comment:
is it required?

Reply:
no its not technically required but the sample snippet in the README has the `  --report_to="wandb"` option